homeassistant/components/media_player/__init__.py

```diff
@@ -595,7 +595,7 @@ def entity_picture(self):
     @property
     def state_attributes(self):
         """Return the state attributes."""
-        if self.state == STATE_OFF:
+        if self.state in [STATE_OFF, STATE_UNKNOWN]:
```

Comment:
I can revert this change and set the status of an unreachable sonos player to `STATE_OFF`. 

What do you think?